content/Rails - Multiple Databases.md

```diff
@@ -0,0 +1,124 @@
+# ** Rails 6 - Multiple Databases **
+
+![main image](images/databases.jpg)
+
+Rails 6.0 came up with this amazing enhancement, but for most people, I'm aware they think that is a new feature.
+In my opinion, both are correct, because the actual state of multiple databases before rails 6.0 was not even useful to consider it a completed feature.
```

Comment:
```suggestion
In my opinion, both are correct, because the actual state of multiple databases before rails 6.0 was not even useful to consider it a completed feature.
```